projects/step-core/src/lib/modules/basics/shared/is-touch-device.token.ts

```diff
@@ -0,0 +1,13 @@
+import { inject, InjectionToken } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+
+export const IS_TOUCH_DEVICE = new InjectionToken<boolean>('Returns true in case of touch device', {
+  providedIn: 'root',
+  factory: () => {
+    const document = inject(DOCUMENT);
+    return (
+      'ontouchstart' in (document?.defaultView || {}) ||
+      !!(document?.defaultView?.navigator as any)?.['msMaxTouchPoints']
```

Comment:
```
declare global {
  interface Navigator {
    msMaxTouchPoints?: number;
  }
}

export const IS_TOUCH_DEVICE = new InjectionToken<boolean>('Returns true in case of touch device', {
  providedIn: 'root',
  factory: () => {
    const document = inject(DOCUMENT);
    return (
      'ontouchstart' in (document?.defaultView || {}) ||
      !!document?.defaultView?.navigator?.['msMaxTouchPoints']
    );
  },
});
```

Reply:
Not relevant as `IS_TOUCH_DEVICE` has been removed